src/styles/blocks/card.scss

```diff
@@ -0,0 +1,95 @@
+@use "../utils/variable" as *;
+
+.card {
+  width: 200px;
+  padding: 16px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+
+  background-color: $bg-color-white;
+  border: 1px solid $border-card;
+  border-radius: 5px;
+
+  &:not(:nth-last-child(-n + 4)) {
+    margin-bottom: 46px;
+  }
+
+  &:not(:nth-child(4n)) {
+    margin-right: 48px;
+  }
```

Comment:
Don't add margin to a BEM block
<img width="987" alt="image" src="https://user-images.githubusercontent.com/99403198/222421683-b60c06ea-8467-443d-814e-442a07cc4209.png">

Reply:
![image](https://user-images.githubusercontent.com/12642122/222457007-0110e8a5-39b3-4bdf-8b36-05cd877a1612.png)